modules/reactor/doc/content/source/meshgenerators/SubdomainExtraElementIDGenerator.md

```diff
@@ -4,9 +4,14 @@
 
 ## Overview
 
-The `SubdomainExtraElementIDGenerator` assigns element IDs based on mesh subdomain IDs of an input mesh.
-If the element IDs do not exist in the input mesh, they will be added by this mesh generator.
+The `SubdomainExtraElementIDGenerator` assigns extra element IDs based on mesh subdomain IDs of an input mesh.
+If the extra element IDs do not exist in the input mesh, this mesh generator will add them.
+The [!param](/Mesh/SubdomainExtraElementIDGenerator/subdomains) parameter can accept both subdomain names and subdomain IDs.
+The [!param](/Mesh/SubdomainExtraElementIDGenerator/extra_element_ids) parameter is a two-dimensional vector separated by semicolon and then white space as normal MOOSE input syntax for two-dimensional vectors.
```

Comment:
I think the easier way to convey this is to add a ## Example input syntax section at the end of the docs before the parameters

Reply:
but this text is welcome too

Reply:
I will add a `!listing`  here.

Reply:
https://mooseframework.inl.gov/docs/PRs/19530/site/source/meshgenerators/SubdomainExtraElementIDGenerator.html

Reply:
looks good